HACKING.md

```diff
@@ -0,0 +1,95 @@
+# Setting up the image
+
+We recommend using [the images](https://aiyprojects.withgoogle.com/voice) we
+provide. Those images are based on [Raspbian](https://www.raspberrypi.org/downloads/raspbian/),
+with a few customizations and are tested on the Raspberry Pi 3. If you prefer
+to setup Raspbian yourself, there are some manual steps you need to take.
+
+## Installing the dependencies
+
+First, make sure you have `git` installed and clone this repository in
+`~/voice-recognizer-raspi`:
+
+```shell
+sudo apt-get install git
+cd
+git clone https://github.com/google/aiyprojects-raspbian.git voice-recognizer-raspi
+```
+
+Then, install the project dependencies and setup the services:
+
+``` shell
+cd ~/voice-recognizer-raspi
+scripts/install-deps.sh
+sudo scripts/install-services.sh
+```
+
+## Installing the Voice HAT driver and config
+
+To use the Voice HAT, you'll need to upgrade your kernel to 4.9, then adjust the
```

Comment:
This is no longer true as Raspbian ships with 4.9 since 2017-07-05. Please remove the apt-get commands and add to the text something like "Your kernel needs to be 4.9 or later. This is available on Raspbian 2017-07-05 and later."